pkgs/tools/security/tracee/default.nix

```diff
@@ -59,27 +56,33 @@ buildGoModule rec {
   # see passthru.tests.integration
   doCheck = false;
 
+  outputs = [ "out" "bin" "legacy" "lib" "share" ];
+
   installPhase = ''
     runHook preInstall
 
-    mkdir -p $out/{bin,share/tracee}
+    mkdir -p $out/{bin,share} $bin/bin $legacy/bin $lib/lib/tracee $share/share/tracee
 
-    mv ./dist/tracee-{ebpf,rules} $out/bin/
+    mv ./dist/tracee $bin/bin/
+    # tracee-ebpf and tracee-rules are being replaced with the singular tracee
+    # kept to match upstream while users migrate across
+    mv ./dist/tracee-{ebpf,rules} $legacy/bin/
+    mv ./dist/tracee.bpf.core.o $lib/lib/tracee/
+    mv ./cmd/tracee-rules/templates $share/share/tracee/
 
-    mv ./dist/rules $out/share/tracee/
-    mv ./cmd/tracee-rules/templates $out/share/tracee/
+    ln -s $bin/bin/tracee $out/bin
+    ln -s $legacy/bin/tracee-{ebpf,rules} $out/bin
+    ln -s $share/share/tracee $out/share/tracee
 
     runHook postInstall
   '';
 
-  doInstallCheck = true;
+  doInstallCheck = false;
```

Comment:
Isn't this the default?

Reply:
yeah, this is left over from debugging where I knew they changed the names of some of the binaries & didn't want to fail the build before I could modify.
Need to revert